base/doc/clsguide.tex

```diff
@@ -977,6 +977,30 @@ \subsection{Passing options around}
 handler, but this handler is not used for |landscape| because that option is
 explicitly declared.
 
+\subsection{Testing dates of other code}
+
+\begin{decl}
+  |\IfPackageAtLeast(TF)| \arg{package} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfClassAtLeast(TF)| \arg{class} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfFileAtLeast(TF)| \arg{file} \arg{date} \arg{true code} \arg{false code}
+\end{decl}
+There are occasions when you need to test if a package is
+at least a particular date (assuming it has been loaded). This can be done using the
+commands |\IfPackageAtLeastTF|, etc. Separate commands are available to test
+packages, classes and general files: in all cases, they rely on the existence
+of a |\Provides...| line in the file being queried. Three versions
+of each command are available, ending |T|, |F| and |TF|, which indicate the
+outcome branches required. Thus for example |\IfPakcageAtLeastT| does not need
+a \meta{false code} argument. The \meta{date} argument can be given either in
+ISO (|YYYY-MM-DD|) or traditional \LaTeX{} (|YYYY/MM/DD|) formats.
```

Comment:
In all other three places of [`clsguide.tex`](https://github.com/latex3/latex2e/blob/1f18053be5081bbe8b4a45193d26e101eaeac4f6/base/doc/clsguide.tex#L392-L394) where the date format is mentioned, only the `YYYY-MM-DD` format is mentioned and it's typeset in `\textsc{yyyy-mm-dd}`. See the first place,

https://github.com/latex3/latex2e/blob/1f18053be5081bbe8b4a45193d26e101eaeac4f6/base/doc/clsguide.tex#L392-L394

Reply:
I can change that - but I think we do need to say you can use the traditional form too